signac_dashboard/modules/schema_display.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+from flask import render_template
+from markupsafe import escape
+
+from signac_dashboard.module import Module
+from signac_dashboard.util import ellipsis_string
+
+
+class SchemaView(Module):
+    """Displays the project schema.
+
+    Long values can be optionally truncated.
+
+    :param max_chars: Truncation length for document values (default:
+        :code:`None`).
+    :type max_chars: int
+    """
+
+    def __init__(
+        self,
+        name="Project Schema",
+        context="ProjectContext",
+        template="cards/document_list.html",
```

Comment:
A more generic name for this file might be appropriate. e.g. `key_value_list.html`. That way it's clear that multiple modules might rely on it (rather than only `DocumentList`).